menas/ui/css/style.css

```diff
@@ -44,25 +44,25 @@ html, body {
 }
 
 .monitoringTitle {
-  textAlign: Center;
+  text-align: Center;
```

Comment:
This was previously ignored, and I must say I liked it more, but even now it breaks nothing. 👍